src/Game/ResearchGUI.h

```diff
@@ -122,17 +125,43 @@ extern bool rmGUIActive;
 
 extern LRUPicture pictures[RM_TOTALPICS];
 
+typedef void (*setPrintListVarCback)(char *directory, char *field, void *printList, TechStatics *techstat, udword *itemIndexPtr);
+
+typedef struct
+{
+    char *name;
+    setPrintListVarCback setVarCB;
+    TechStatics *techstat;
+    void *dataPtr;
+} scriptEntryPrintList;
```

Comment:
- `void *dataPtr` => `TechPrintList *techPrintList`
- Any possibilities of hiding these from consumers if the logic implementation can be moved into `ResearchGUI.c`?